### ISSUE TITLE
Codeclimate fails using Credo 1.0

### ISSUE THREAD

**Issue by Eptis** [fazibear/codeclimate-credo#14]
If I run codeclimate with the latest credo (v1.0) I get the error:
```
error: (CC::CLI::Analyze::EngineFailure) engine credo failed with status 1 and stderr
** (UndefinedFunctionError) function Credo.Check.Readability.AliasOrder.base_priority/0 is undefined (module Credo.Check.Readability.AliasOrder is not available)
```

Any idea how I can make a PR to update the codeclimate-credo plugin to include the latest Credo checks?

**Comment by fazibear** [comment]
I'll take a look at this. This version is not the final. There are PR waiting for merge on the credo side.

Anyway is standalone credo works fine for your project?

**Comment by Eptis** [comment]
Yup standalone credo works fine. Its not so much the version that is incompatible, but the specific modules in those versions that are not present in codeclimate. Commenting those out works :) Let me know if I can contribute anything!

**Comment by fazibear** [comment]
Because credo guy doesn't respond to any of my PRs, I've forked it here
https://github.com/fazibear/credo/tree/codeclimate and patch it.
During the build, it fetches code from there. It may be out of date.

**Comment by maxjacobson** [comment]
@fazibear Hi there. I work at Code Climate. Anything I can do to help get this next release out?

**Comment by davehenton** [comment]
Hey @fazibear, Dave here with Code Climate -- just checking in here. Let me know if there's anything I can do to help with this next release. Thanks!

**Comment by davehenton** [comment]
hey @fazibear - just checking in here. Let me know how I can help out. Thanks!

**Comment by larkinscott** [comment]
Hey @fazibear - I work at Code Climate. I went ahead and forked this repo over to our community org and marked you as a collaborator. Hope that's ok! If so, can we archive this repo?
